src/Components/Track/Track.js

```diff
@@ -0,0 +1,87 @@
+import React, { useContext } from 'react';
+import { PlaylistContext } from '../Context/PlaylistContext';
+import './Track.css';
+
+export default function Track(props) {
+
+    const infoContext = useContext(PlaylistContext);
+
+    function trackAction(track) {
+        if (props.button === '+') {
```

Comment:
Instead of using this logic, another logic could be

A Track is already aware of infoContext.playlist. So a Track knows if it's on the playlist or not. This means that the Track component can decide whether or not it can add or remove itself from the playlist. This means that you can also remove tracks from the playlist from the search results and you don't have to pass around the props.

If keeping this logic, I would change the `button` prop to something else. Doesn't feel right to pass + and - down components. Also, create a CONSTANT variable for the values.